src/asymptotics.jl

```diff
@@ -121,15 +121,15 @@ function _α_αp_poly_10(v, x::T) where T
     μ = 4 * v^2
     s0 = one(T)
     s1 = (1 - μ) / 8
-    s2 = evalpoly(μ, T.((-0.1953125, 0.203125, -0.0078125)))
-    s3 = evalpoly(μ, T.((1.0478515625, -1.1591796875, 0.1123046875, -0.0009765625)))
-    s4 = evalpoly(μ, T.((-11.466461181640625, 13.1358642578125, -1.71624755859375, 0.0469970703125, -0.000152587890625)))
-    s5 = evalpoly(μ, T.((211.27614974975586, -246.8455924987793, 37.067405700683594, -1.5151596069335938, 0.017223358154296875, -2.6702880859375e-5)))
-    s6 = evalpoly(μ, T.((-5892.0457146167755, 6965.548173427582, -1128.6143367290497, 56.11658954620361, -1.0105445384979248, 0.005837917327880859, -5.0067901611328125e-6)))
-    s7 = evalpoly(μ, T.((231884.6359563172, -276225.9131319225, 46889.68168082833, -2607.1347218453884, 59.29277476668358, -0.5644365847110748, 0.0018794238567352295, -9.834766387939453e-7)))
-    s8 = evalpoly(μ, T.((-1.225106763286615e7, 1.4670878180751745e7, -2.5692206274838205e6, 153362.43350451812, -4001.4928083932027, 49.41718461737037, -0.2788656409829855, 0.0005833245813846588, -1.9976869225502014e-7)))
-    s9 = evalpoly(μ, T.((8.369564497799761e8, -1.0061560971482614e9, 1.80175839935865e8, -1.1293844119552301e7, 322272.3393258663, -4655.44006793818, 34.77845393284224, -0.12591543863527477, 0.0001761710154823959, -4.1618477553129196e-8)))
-    s10 = evalpoly(μ, T.((-7.181611223268327e10, 8.659091218989731e10, -1.576853447634526e10, 1.0243675365394899e9, -3.1127612822052997e7, 499001.19046756154, -4427.281508191307, 21.557856538012857, -0.053067848184582544, 5.209072696743533e-5, -8.843926480039954e-9)))
+    s2 = evalpoly(μ, (-0.1953125, 0.203125, -0.0078125))
```

Comment:
You could keep the conversion to `T` by writing explicitly
```suggestion
    s2 = evalpoly(μ, (T(-0.1953125), T(0.203125), T(-0.0078125)))
```
This is actually what I had in mind - it avoids broadcasting and hence should improve compile time but shouldn't change any results or dispatches.

Reply:
O interesting! Let me check that out. (or let me see if there is an easy way to automate that 😆)